schema/filespec-schema.json

```
@@ -130,7 +130,10 @@
         "default": "false"
       },
       "sortBy": {
-        "type": "string",
+        "type": "array",
+        "items": {
+          "type": "string"
+        },
         "description": "A list of semicolon-separated fields to sort by. The fields must be part of the 'items' AQL domain.",
```

Let's fix also the description
```suggestion
        "description": "A list of fields to sort by. The fields must be part of the 'items' AQL domain.",
```